host_core/test/e2e/controlinterface_test.exs

```diff
@@ -1,15 +1,14 @@
 defmodule HostCore.E2E.ControlInterfaceTest do
   use ExUnit.Case, async: false
 
-  @echo_key "MADQAFWOOOCZFDKYEYHC7AUQKDJTP32XUC5TDSMN4JLTDTU2WXBVPG4G"
-
-  @kvcounter_key "MCFMFDWFHGKELOXPCNCDXKK5OFLHBVEWRAOXR5JSQUD2TOFRE3DFPM7E"
-  @redis_key "VAZVC4RX54J2NVCMCW7BPCAHGGG5XZXDBXFUMDUXGESTMQEJLC3YVZWB"
-  @redis_link "default"
-  @redis_contract "wasmcloud:keyvalue"
+  @echo_key HostCoreTest.Constants.echo_key()
+  @kvcounter_key HostCoreTest.Constants.kvcounter_key()
+  @redis_key HostCoreTest.Constants.redis_key()
+  @redis_link HostCoreTest.Constants.default_link()
+  @redis_contract HostCoreTest.Constants.keyvalue_contract()
 
   test "can get claims" do
-    {:ok, bytes} = File.read("test/fixtures/actors/echo_s.wasm")
+    {:ok, bytes} = File.read(HostCoreTest.Constants.echo_path())
```

Comment:
It seems inconsistent to use the module constants on lines 4-8 but to call this function directly. For consistency, shouldn't we be using something like `@echo_path` here?

Reply:
Yep, I'll refactor this